src/desktops/dskHostGame.cpp

```diff
@@ -431,6 +431,7 @@ void dskHostGame::UpdatePlayerRow(const unsigned row)
  */
 void dskHostGame::Msg_PaintBefore()
 {
+    Desktop::Msg_PaintBefore();

```

Comment:
Yep. For 40 classes or so. Its just boring ;)
I see if I can make a regex or so to do this automatically. We need it for the gameobjects too to get rid of all the `Destroy_xxx` functions and the like